lib/src/views/Login.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <v-container class="mt-10">
+        <v-card 
+            class="mx-auto"
+            max-width="550"
+            max-height="600"
+        >
+            <div class="text-h2 pl-2 pt-2">Sign in</div>
+
+            <v-container>
+                <v-row class="ml-2 mt-5">
+                    <v-icon>mdi-account-circle</v-icon>
+                    <v-col cols="12" sm="6" md="10 mt-1">
+                        <v-text-field  dense label="Email">
+                        </v-text-field>
+                    </v-col>
+                </v-row>
+                <v-row class="ml-2">
+                    <v-icon>mdi-lock</v-icon>
+                    <v-col cols="12" sm="6" md="10 mt-1">
+                        <v-text-field dense :type="type" label="Password"></v-text-field>
+                    </v-col>
+                        <v-icon @click="showPassword" v-if="type=='password'">mdi-eye-off</v-icon>
```

Comment:
I think it would be cleaner to interpolate mdi-eye-off if password is set as the type otherwise use mdi-eye:
```suggestion
                        <v-icon @click="showPassword">{{ `mdi-eye-${type=='password' ? 'off' : ''}` }}</v-icon>
```